src/site/stages/build/process-cms-exports/transformers/node-person_profile.js

```diff
@@ -1,27 +1,38 @@
-const { getDrupalValue } = require('./helpers');
+const { getDrupalValue, isPublished, utcToEpochTime } = require('./helpers');
 
-const transform = entity => ({
+const transform = (entity, { ancestors }) => ({
   entityType: 'node',
   entityBundle: 'person_profile',
   title: `${getDrupalValue(entity.fieldNameFirst)} ${getDrupalValue(
     entity.fieldLastName,
   )}`,
+  entityPublished: isPublished(getDrupalValue(entity.moderationState)),
   fieldBody: getDrupalValue(entity.fieldBody),
   fieldDescription: getDrupalValue(entity.fieldDescription),
   fieldEmailAddress: getDrupalValue(entity.fieldEmailAddress),
   fieldLastName: getDrupalValue(entity.fieldLastName),
   fieldMedia: entity.fieldMedia.length > 0 ? entity.fieldMedia[0] : null,
   fieldNameFirst: getDrupalValue(entity.fieldNameFirst),
-  fieldOffice: entity.fieldOffice[0]
-    ? {
-        entity: {
-          entityLabel: entity.fieldOffice[0].entity.entityLabel,
-          entityType: entity.fieldOffice[0].entity.entityType,
-        },
-      }
-    : null,
+  // Check if fieldOffice is an ancestor of this entity
+  // If so, ignore it, because we'll ignore it in parent transformer
```

Comment:
It's probably worth calling out the bit where if the `entity.fieldOffice[0]` is in the `ancestors`, it'll be un-transformed, but we need it to be transformed in _this_ transformer or it'll throw an error. Since the office is an ancestor, we know we're going to ignore it once we get around to transforming the parent office, so we're setting it to `null` to avoid errors.

If you can figure out how to shorten that, that'd be awesome! :sweat_smile: Basically, I think it's worth calling out for future engineers to understand why we needed to "ignore" the parent `fieldOffice` in here.

Reply:
Looks like this change is also [in @jhonnyoddball's PR](https://github.com/department-of-veterans-affairs/vets-website/pull/13827#discussion_r469541785). Not sure which one it should be changed in. I'll let you and Jhonny hash that out. :sweat_smile: :see_no_evil: